templates/CRM/Eventsextras/Form/Settings.tpl

```diff
@@ -0,0 +1,63 @@
+{*
+ +--------------------------------------------------------------------+
+ | CiviCRM version 5                                                  |
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC (c) 2004-2018                                |
+ +--------------------------------------------------------------------+
+ | This file is a part of CiviCRM.                                    |
+ |                                                                    |
+ | CiviCRM is free software; you can copy, modify, and distribute it  |
+ | under the terms of the GNU Affero General Public License           |
+ | Version 3, 19 November 2007 and the CiviCRM Licensing Exception.   |
+ |                                                                    |
+ | CiviCRM is distributed in the hope that it will be useful, but     |
+ | WITHOUT ANY WARRANTY; without even the implied warranty of         |
+ | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.               |
+ | See the GNU Affero General Public License for more details.        |
+ |                                                                    |
+ | You should have received a copy of the GNU Affero General Public   |
+ | License and the CiviCRM Licensing Exception along                  |
+ | with this program; if not, contact CiviCRM LLC                     |
+ | at info[AT]civicrm[DOT]org. If you have questions about the        |
+ | GNU Affero General Public License or the licensing of CiviCRM,     |
+ | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
+ +--------------------------------------------------------------------+
+*}
+<div class="crm-block crm-form-block crm-events-extras-form-block">
+  <div class="crm-submit-buttons">
+    {include file="CRM/common/formButtons.tpl" location="bottom"}
+  </div>
+  <h3>{ts}Front Page Settings{/ts}</h3>
```

Comment:
I was thinking that we can make the sections array a bit more better so we have just one loop here.
e.g

```php
$section['frontpageConfig'] = [
'section_name' => 'Front Page Settings'
'fields' => [``````]
]

```
Or some better structure so we will not have to do three similar loops but one loop to display the form elements. 
What do you think?

Reply:
@tunbola   We will implement it next PR as form will be updated for default options.

Reply:
We can actually implement it now because the change requested will not affect the default options, it only makes the present code better, except you have some other reasons not to do that?